packages/jellyfish-api-core/__tests__/category/loan/setLoanToken.test.ts

```diff
@@ -0,0 +1,258 @@
+import { LoanMasterNodeRegTestContainer } from './loan_container'
+import BigNumber from 'bignumber.js'
+import { Testing } from '@defichain/jellyfish-testing'
+import { GenesisKeys } from '@defichain/testcontainers'
+
+describe('Loan', () => {
+  const container = new LoanMasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should setLoanToken', async () => {
+    const priceFeedId = await testing.container.call('appointoracle', [await testing.generateAddress(), [{
+      token: 'Token1',
+      currency: 'USD'
+    }], 1])
+    await testing.generate(1)
+
+    const loanTokenId = await testing.rpc.loan.setLoanToken({
+      symbol: 'Token1',
+      name: 'Token1',
+      priceFeedId
+    })
+    expect(typeof loanTokenId).toStrictEqual('string')
+    expect(loanTokenId.length).toStrictEqual(64)
+    await testing.generate(1)
+
+    const data = await testing.container.call('listloantokens', [])
+    expect(data).toStrictEqual({
+      [loanTokenId]: {
+        token: {
+          1: {
+            symbol: 'Token1',
+            symbolKey: 'Token1',
+            name: 'Token1',
+            decimal: 8,
+            limit: 0,
+            mintable: false,
```

Comment:
This value should be true. See https://cakedefi.slack.com/archives/G01LSAS1KRQ/p1630980439223400?thread_ts=1630947127.222800&cid=G01LSAS1KRQ

But anyway, kindly ignore this wrong boolean flag. I will change the code once there is update in the C++ side

Reply:
You could try expecting any boolean for now since we know the value will change

Reply:
Never mind, I will replace the docker image will the latest one quite frequently. And if they fix it, then the test will run failed. And I shall fix accordingly.

Reply:
See https://github.com/DeFiCh/jellyfish/pull/568#discussion_r705105016

As I set the default value in Javascipt, the returned mintable is always true now.